src/pages/Register.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import register from '../assets/styles/pages/register.css'
+import {FaUserAlt} from "react-icons/fa";
```

Comment:
It's appropriate to use spaces between elements

For Example:

import { FaUserAlt } .........